src/app/client/js/index.jsx

```diff
@@ -18,11 +18,11 @@ render(
   <HashRouter>
     <Switch>
       <Route exact path="/">
-        <Redirect to="/pattern/client" />
+        <Redirect to="/pattern/server" />
       </Route>
       <Route path="/pattern/:pattern" component={App} />
       <Route path="*">
-        <Redirect to="/pattern/client" />
```

Comment:
This catch all route will also redirect `/patterns/client` so no need to add special logic